1회차/이상욱/1_모음이보이지않는사람.py

```diff
@@ -1,3 +1,16 @@
 import sys
 
 sys.stdin = open("_모음이보이지않는사람.txt")
+
+word_list = ['a', 'e', 'i', 'o', 'u']
+T = int(input())
+
+for i in range(1, T+1):
+    word = input()
+
+    for j in range(len(word)):
+        if word[j] in word_list:
+           word = word.replace(word[j], ' ')
+    word =word.replace(' ', '')
+    print(f'#{i} {word}')
+
```

Comment:
`replace` 함수 활용이 아주 좋네요! 👍 👍 
`replace` 할 때 공백이 아닌 바로 빈 문자열로 바꿔주셔도 됩니다!
```suggestion
    for j in range(len(word)):
        if word[j] in word_list:
           word = word.replace(word[j], '')
    print(f'#{i} {word}')

```